### PR TITLE
make pricing details more explicit

### PR DESCRIPTION
**Pull request by sebastian-lerner**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by sebastian-lerner** [comment]
@rosieyohannan hi, any chance you can review when you get a sec?

**Comment by sebastian-lerner** [comment]
@thomasr8 can you give this a review when you get a chance please?

File: jekyll/_cci2/ip-ranges.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Enable CircleCI jobs to go through a set of well-defined IP address ranges.
 
 IP ranges is a feature for CircleCI customers who need to configure IP-based access to their restricted environments. As part of this feature, CircleCI provides a list of well-defined IP address ranges associated with the CircleCI service. CircleCI jobs that have this feature enabled will have their traffic routed through one of the defined IP address ranges.
 
-The feature is currently available in preview to customers on a [Performance, Custom, or Scale plan](https://circleci.com/pricing/).
+The feature is currently available in preview to customers on a [Performance, Custom, or Scale plan](https://circleci.com/pricing/). Pricing will be calculated based on network data usage of jobs that have opted in to using the IP ranges feature.  There is no charge while the feature is in preview and pricing details will be shared once the feature is generally available.
```
Comment (thomasr8) [inline jekyll/_cci2/ip-ranges.md:24]:
```suggestion
The feature is currently available in preview to customers on a [Performance, Custom, or Scale plan](https://circleci.com/pricing/). Pricing will be calculated based on network data usage of jobs that have opted in to using the IP ranges feature. There is no charge while the feature is in preview and pricing details will be shared once the feature is generally available.  
```

**Review by thomasr8 — APPROVED** [review]
Just one very minor suggestion. Looks good!

**Comment by sebastian-lerner** [comment]
@thomasr8 thank you! could we merge it today if possible?

**Pull request merged**